lib/checks/aria/aria-allowed-role.js

```diff
@@ -3,6 +3,7 @@
  * https://www.w3.org/TR/html-aria/#docconformance
  * https://www.w3.org/TR/SVG2/struct.html#implicit-aria-semantics
  */
+const { dom } = axe.commons;
```

Comment:
Please add an "inapplicable" message to the check JSON so that we communicate with the user what's going on.

Reply:
@WilcoFiers 

I have gone with below. Let me know if you have suggestions.

`"incomplete": "Element is not visible, and ARIA role{{=it.data && it.data.length > 1 ? 's' : ''}} {{=it.data.join(', ')}} {{=it.data && it.data.length > 1 ? 'are' : ' is'}} not allowed for given element"`

Reply:
@marcysutton - any suggestions for the incomplete message - https://github.com/dequelabs/axe-core/pull/1225/files#diff-2f072a5102050cd26c55026d5d9f9c4fR13